### DISCUSSION TITLE
Versioned URL for HTML Component?

### DISCUSSION THREAD

**Discussion by dbrian57** [stoplightio/elements#2172] [Q&A] [answered]
Hi, are there versioned URLs for the HTML component of Elements? i.e. if Elements pushes out a new change, is there a URL somewhere that reflects the prior version of the JS file? We want to use Elements to host our docs, but we'd prefer the stability of using a version that we know works with our site and then upgrade as needed. Thanks for any help on this!

`<script src="https://unpkg.com/@stoplight/elements/web-components.min.js"></script>`

**Comment by babekasadli** [comment, marked as answer]
There is currently no such feature. However, you can update the item link in JS using `HTMLElement.innerText`. I hope you asked that
